utils.py

```diff
@@ -0,0 +1,6 @@
+def clamp(num: int, min_value: int, max_value: int):
```

Comment:
We didn't need this, we will leave it, might be useful in the future but... well :D